src/Assignment-2/assignment2.css

```diff
@@ -1,7 +1,54 @@
 /* Our CSS file. */
 
-
 .container, .content {
-width: 100%;
-height: 100%;
-}
+    width: 100%;
```

Comment:
no use of ids you have created , its like declaring a variable and not using it.